src/DataSource.ts

```diff
@@ -31,4 +31,30 @@ export class DataSource extends DataSourceWithBackend<GAQuery, GADataSourceOptio
         : [];
     });
   }
+
+  async getDimensions(): Promise<Array<SelectableValue<string>>> {
+    return this.getResource('dimensions').then(({ dimensions }) => {
+      return dimensions
+        ? dimensions.map((dimension: any) => {
+            return {
+              label: dimension.id + ' ' + dimension.attributes.description,
```

Comment:
![image](https://user-images.githubusercontent.com/38449269/110846161-cee6ed80-82ee-11eb-9d6d-18a6b5069ff6.png)
As shown in the picture above, long description breaks the ui
It seems better to use uiName or any other good idea
```suggestion
              label: dimension.id + ' ' + dimension.attributes.uiName,
```
p.s [ga api metadata](https://www.googleapis.com/analytics/v3/metadata/ga/columns?pp=1)